lms/djangoapps/teams/static/teams/js/spec/topic_card_spec.js

```diff
@@ -0,0 +1,41 @@
+(function (define) {
+    'use script';
+
+    define(['jquery',
```

Comment:
I don't think you need to wrap this with the ``define`` variable as in Jasmine tests there is no ``RequireJS`` namespace defined. I'm not 100% sure of this, but I hope it is true. :-)